src/translation/translation_de_DE.ts

```diff
@@ -4140,7 +4140,7 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../util.h" line="648"/>
         <source>Directory server list full</source>
-        <translation type="unfinished"></translation>
+        <translation>Verzeichnis voll</translation>
```

Comment:
Mmm, not sure on this one... `Verzeichnisserverliste voll`?  It's the server list that's full, that server list belongs to the directory.

Reply:
Yes. That would work. Probably Verzeichnis Server-Liste is better.